config/crd/bases/jobset.x-k8s.io_jobsets.yaml

```diff
@@ -9492,13 +9492,8 @@ spec:
                       type: object
                   required:
                   - name
+                  - template
                   type: object
-                  x-kubernetes-validations:
```

Comment:
is this validation removal supposed to be part of this PR? I thought that was done in #63

Reply:
Hmm.  it shouldn't be here but all I did was `make generate manifests` and then it added this?

Reply:
https://github.com/kubernetes-sigs/jobset/pull/63/files doesn't seem to have the manifests files committed so maybe thats why they are included in mine?

Reply:
ah that must be it, ok cool